tests/testthat/test-kobo.R

```diff
@@ -100,3 +100,82 @@ test_that("non-existing kobo host throws error", {
   kobo <- Kobo$new(base_url_v2 = "https://nokobohere.correlaid.org", kobo_token = Sys.getenv("KBTBR_TOKEN"))
   expect_error(kobo$get("assets/"), regexp = "^SSL.+certificate.+")
 })
+
+
+#' -----------------------------------------------------------------------------
+#' Testing POST methods
+test_that("Kobo$post can clone assets", {
+    vcr::use_cassette("kobo-clone-assets-simple-post", {
+        kobo <- Kobo$new(base_url_v2 = BASE_URL, kobo_token = Sys.getenv("KBTBR_TOKEN"))
+        clone_asset <- kobo$post("assets/", body = list("clone_from" = "a84jmwwdPEsZMhK7s2i4SL",
+                                                        "name" = "vcr_test_name",
+                                                        "asset_type" = "survey"))
+    })
+    expect_equal(clone_asset$url,"https://kobo.correlaid.org/api/v2/assets/")
+    expect_equal(clone_asset$method,"post")
+    expect_equal(clone_asset$status_code,201)
+    expect_equal(clone_asset$success(),TRUE)
+    expect_equal(clone_asset$status_http()$message,"Created")
+    expect_equal(clone_asset$status_http()$explanation,"Document created, URL follows")
+})
+
+test_that("kobo$clone_asset can clone assets", {
+    vcr::use_cassette("kobo-post-clone-asset", {
+        kobo <- suppressWarnings(Kobo$new(base_url_v2 = BASE_URL, kobo_token = Sys.getenv("KBTBR_TOKEN")))
```

Comment:
it actually dines't help. I wanted to avoid the messages about api version during tests but they were still returned

Reply:
Okay! In that case you could use `suppressMessages()`  for that purpose! In general, I would argue to avoid suppressing warnings inside tests, as we could miss some other warnings (unintended).